Sources/Widgets/Widgets3D/ResliceCursorWidget/behavior.js

```diff
@@ -68,6 +68,43 @@ export default function widgetBehavior(publicAPI, model) {
     model.widgetState.deactivate();
   };
 
+  publicAPI.handleStartMouseWheel = (callData) => {
+    publicAPI.handleMouseWheel(callData);
+  };
+
+  publicAPI.handleMouseWheel = (calldata) => {
+    const direction = calldata.spinY;
+    const currentCenter = model.widgetState.getCenter();
+    const imageSpacing = model.widgetState.getImage().getSpacing();
+
+    // Get the normal use to translate the center along
+    const normal = calldata.pokedRenderer
+      .getRenderWindow()
+      .getRenderers()[0]
+      .getActiveCamera()
+      .getDirectionOfProjection();
+
+    // Direction of the projection if the inverse of what we want
+    const translationVector = vtkMath.multiplyScalar(normal, -1);
+
+    const newCenter = [
+      currentCenter[0] + direction * translationVector[0] * imageSpacing[0],
+      currentCenter[1] + direction * translationVector[1] * imageSpacing[1],
+      currentCenter[2] + direction * translationVector[2] * imageSpacing[2],
+    ];
+
+    model.widgetState.setCenter(newCenter);
+    updateState(model.widgetState);
+
+    publicAPI.invokeInteractionEvent();
+
+    // When interacting, plane actor and lines must be re-rendered on other views
+    publicAPI.getViewWidgets().forEach((viewWidget) => {
+      viewWidget.getInteractor().requestAnimation(publicAPI);
```

Comment:
why don't you do that with the other events ?

Reply:
shouln't you also "cancelAnimation" also ?

Reply:
I finally find out whats happening. When we start scrolling, we want to request animation. Each time we scroll, we animate. At the end of the scrolling, we cancel all animations.
It works better now. But, it doesn't solve the preivous issue where I have to update 3 views here:
`// Note: Need to refresh also the current view because of adding the mouse wheel
        // to change slicer
        // .filter((_, index) => index !== i)`

The scrolling directly affects the current imageReslice. So, we have to update the slice by calling `updateReslice` on it.
Doest it seem clearer now ? The `updateReslice` function update the reslice filter. But only the modified one will be "rerender". So, when I will scroll on view X, only the viewX will be rerender.

Reply:
Do you mean that `updateReslicePlane` will return false for the view X when you do not "scroll" (but rotate line Y) ?

Reply:
Yes! For example, if you rotate the green line on the X view, then it will only call 
`    actor.setUserMatrix(reslice.getResliceAxes());
    widget.resetCamera(renderer, viewtype);` of the Y view.
But, we are agreed that the `updateReslicePlane` method, will be called on each views, each time the widgetState is updated. No matter if you translate, rotate, scroll.